tmail-backend/blob/secondary-blob-store/src/main/scala/com/linagora/tmail/blob/secondaryblobstore/ExtendedMailboxEventSerializer.scala

```diff
@@ -0,0 +1,66 @@
+package com.linagora.tmail.blob.secondaryblobstore
+
+import com.linagora.tmail.blob.secondaryblobstore.FailedBlobEvents.{BlobAddition => JavaBlobAddition}
+import jakarta.inject.Inject
+import julienrf.json.derived
+import org.apache.james.blob.api.{BlobId, BucketName}
+import org.apache.james.event.json.{Event, JsonSerialize, ScalaConverter}
+import org.apache.james.events.{EventSerializer, Event => JavaEvent}
+import org.apache.james.mailbox.model.{MailboxId, MessageId}
+import org.apache.james.mailbox.quota.QuotaRootDeserializer
+import play.api.libs.json.{JsResult, Json, OFormat}
```

Comment:
No please no more playjson for this I 100% prefer we use `james-json` utilities.

CF StateChangeEventDTO & friends

Reply:
These serializers are used by different event buses so I don't they should be merged